tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/SimpleSymmetricClusterTest.java

```diff
@@ -370,6 +371,45 @@ public void testSimple_TwoNodes() throws Exception {
 
    }
 
+   @Test
+   public void testFqqnInitialDistribution() throws Exception {
+      CountDownLatch countDownLatch = new CountDownLatch(1);
+      final String address = "myAddress";
+      final String queue = "myQueue";
+      final String fqqn = CompositeAddress.toFullyQualified(address, queue);
+
+      setupServer(0, false, isNetty());
+      setupServer(1, false, isNetty());
+
+      setupClusterConnection("cluster0", address, MessageLoadBalancingType.ON_DEMAND, 1, isNetty(), 0, 1);
+      setupClusterConnection("cluster1", address, MessageLoadBalancingType.ON_DEMAND, 1, isNetty(), 1, 0);
+
+      startServers(0, 1);
+
+      setupSessionFactory(0, isNetty());
+      setupSessionFactory(1, isNetty());
+
+      createQueue(0, address, queue, null, false);
+      createQueue(1, address, queue, null, false);
+
+      waitForBindings(0, address, 1, 0, true);
+      waitForBindings(0, address, 1, 0, false);
+      waitForBindings(1, address, 1, 0, true);
+      waitForBindings(1, address, 1, 0, false);
+
+      addConsumer(0, 0, fqqn, null);
+      consumers[0].consumer.setMessageHandler((m) -> countDownLatch.countDown());
+
+      waitForBindings(0, address, 1, 1, true);
+      waitForBindings(1, address, 1, 1, false);
+
+      send(1, fqqn, 1, true, null);
```

Comment:
Another test variant to increase the test coverage could send to address, i.e.
```suggestion
      send(1, address, 1, true, null);
```